src/main/java/Task.java

```diff
@@ -0,0 +1,26 @@
+public class Task {
+    private final String text;
+    private boolean isDone;
+
+    public Task(String text) {
+        this.text = text;
+        this.isDone = false;
+    }
+
+    public void markAsDone() {
```

Comment:
Perhaps the setter should be named as setDone(), according to the coding standard?